app/scripts/configs/datatype-to-track-type.js

```diff
@@ -3,7 +3,26 @@ import { TRACKS_INFO } from '.';
 export const DATATYPE_TO_TRACK_TYPE = orientation => {
   const localDatatypeToTrackType = {};
 
-  TRACKS_INFO.filter(x => x.orientation === orientation).forEach(ti => {
+  // can a track be placed in a given orientation
+  const orientationMatches = x => {
```

Comment:
What is `x` here? I assume it's not a number but the track config or something? I'd suggest using a more verbose variable name so it's easier to make sense of what the function is trying to do (testing whether a track can be in a certain orientation I assume)

Reply:
Renamed `x` to  `trackInfo`.